IndicoV2.Abstractions/Submissions/Models/FilterConverter.cs

```diff
@@ -15,12 +16,57 @@ public static ssSubmissionFilter ConvertToSs(this IFilter filter)
         {
             if (filter is v2SubmissionFilter submissionFilter)
             {
-                return new ssSubmissionFilter()
+                var ssFilter = new ssSubmissionFilter();
+                if (submissionFilter.FileType != null)
                 {
-                    InputFilename = submissionFilter.InputFilename,
-                    Retrieved = submissionFilter.Retrieved,
-                    Status = (StrawberryShake.SubmissionStatus?)submissionFilter.Status
-                };
+                    ssFilter.Filetype = submissionFilter.FileType.Select(x => (FileType)Enum.Parse(typeof(FileType), x)).ToList();
+                }
+                if (submissionFilter.InputFilename != null)
```

Comment:
nit but you should be able to do `String.IsNullOrEmpty()` here

similarly for nullable types `?` you can do `if (varname.HasValue()){thing}`

also, I would leave a comment in here somewhere as to why this had to be done, so in the future you don't come back and curse it 😆